app/src/test/java/waiter/HttProtocolTest.java

```diff
@@ -37,14 +42,19 @@ void returnsOkWithNoBodyWhenGetMethodWithExistingUrl() {
     @Test
     void returnsOkWithNoBodyWhenHeadMethodWithExistingUrl() {
```

Comment:
How are these tests any different? What are we testing here?

Reply:
These tests are each testing the response result to a method being used (one each for post, get, head, etc)

Reply:
Wouldn't we want to test that as part of the `Router`? This doesn't seem like it should be the responsibility of the `HttpProtocol`.

Reply:
Would we be able to pair on this/discuss this on zoom at some point (no rush at all), I have some questions/clarifications I think would be easier to talk about over chat 😄